src/app/modules/admin-shared/no-record/no-record.component.html

```diff
@@ -0,0 +1 @@
+<div class="message" *ngIf="recordCount == 0 || !dataSource">No records found</div>
```

Comment:
this condition should be only in corresponding pages

Reply:
yes removed